File: even_odd.java
import java.util.Scanner;
public class even_odd {
    public void display(int n){
        if(n%2==0){
            System.out.println("EVEN");
        }
        else{
            System.out.println("ODD");
        }  
    }
    public static void main(String[] args) {
        Scanner myScanner = new Scanner(System.in);
        System.out.println("enter a number: ");
        int num = myScanner.nextInt();
        even_odd myObj = new even_odd();
        myObj.display(num);
    }   
}

File: even_odd.cpp
#include<iostream>
using namespace std;
class even_odd(){
    public:
    void dislay(int num){
        if(n%2==0){
            cout<<"enter a number:\n"<<num<<"even"<endl;
        }
        else{
            cout<<"enter a number:\n"<<num<<"odd"<<endl;
        }
    }
};
int main(){
        int number;
    even_odd obj1;
    cout<<"enter a number : ";
    cin>>num;
    obj1.check(num);

}
